lib/logstash/filters/translate.rb

```diff
@@ -62,11 +65,20 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
   #     merci: gracias
   #     old version: new version
   #     
-  # NOTE: it is an error to specify both `dictionary` and `dictionary_path`
+  # NOTE: it is an error to specify both `dictionary` and `dictionary_path` or `dictionary_url`
   config :dictionary_path, :validate => :path
 
-  # When using a dictionary file, this setting will indicate how frequently
-  # (in seconds) logstash will check the YAML file for updates.
+  # The full URI path of a Web service who generates an yml format response. 
+  # The response generated needs to be equals than needed for @dictionary_path
+  #
+  # NOTE: it is an error to specify both `dictionary` and `dictionary_path` or `dictionary_url`
+  config :dictionary_url, :validate => :string
+
+  # Filename (without extension) where .yml will be stored from a Web service.
+  config :file_to_download, :validate => :string, :default => "dictionary"
```

Comment:
@angel9484 I agree with @jordansissel, i don't see this as a mandatory parameter for the user. it would be nicer if this could be something as he commented. In ruby this is not complicated to archive, you can for example use http://ruby-doc.org/core-2.1.5/File.html#method-i-flock for the locking.

I love the idea of having simple interfaces for users :+1: 

what do you think?